jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,217 @@
+---
+
+- job:
+    name: osd-testing-nightly-trigger
+    project-type: pipeline
+    description: "Nightly pipeline trigger for QE OSD cluster (rhmi-qe1) testing."
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H(20) 3 * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            CLUSTER_NAME=rhmi-qe1
+            RECIPIENTS=integreatly-qe@redhat.com
+            INSTALLATION_BRANCH=master
+            ROUTER_SHARD=f2d1
+            USERNAME=osdtester
+            TEST_SUITES_REPOSITORY=https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            TEST_SUITES_BRANCH=master
+            GH_CLIENT_ID=836affcd7fd1bcebf9ae
+            GH_CLIENT_SECRET=72e97db6f1a2a6d8d2be66686cf52ccf15b3588f
+            CUSTOMER_ADMIN_PASSWORD=Password1
+            CREDENTIALS_ID=osd-s3-access-key-and-secret
+            THREESCALE_BUCKET_NAME=integreatly-qe1-3scale
+            THREESCALE_BUCKET_REGION=eu-west-1
+            BACKUPS_BUCKET_NAME=rhmi-qe1-backups
+            ADMIN_USERNAME=admin01
+            ADMIN_PASSWORD=Password1
+            CLEAN_RESOURCES=true
```

Comment:
would it make sense to have this set to 'false', so one can check during the day?